configs/components/openssl.rb

```diff
@@ -54,6 +66,17 @@
     ldflags = ''
     cflags = "$${CFLAGS} -static-libgcc"
     pkg.environment "CC" => "/opt/pl-build-tools/bin/gcc"
+  elsif platform.is_windows?
+    target = platform.architecture == "x64" ? "mingw64" : "mingw"
+    pkg.environment "PATH" => "#{settings[:gcc_bindir]}:$$PATH"
+    pkg.environment "CYGWIN" => settings[:cygwin]
+    pkg.environment "CC" => settings[:cc]
+    pkg.environment "CXX" => settings[:cxx]
+    make = "/usr/bin/make"
```

Comment:
Minor issue - my preference would be to group all the pkg.environment statements together and put the make reassignment after them. This is an odd-case, but wondering should you put the make assignment back to line 70 and follow it with pkg.environment for the make (if you think this is OCD speaking, feel free to ignore).